sdk/servicebus/azure-servicebus/azure/servicebus/management/_utils.py

```diff
@@ -320,6 +330,15 @@ def _validate_topic_subscription_and_rule_types(
             )
         )
 
+
+def _normalize_forward_to_address(forward_to, fully_qualified_namespace):
+    # type: (str, str) -> str
+    if forward_to:  # for python 2.7, cannot urlparse None
+        parsed = urlparse.urlparse(forward_to)
+        forward_to = "sb://" + fully_qualified_namespace + "/" + forward_to if not parsed.netloc else forward_to
+    return forward_to
```

Comment:
I would probably rename the method to be more general and tweak the logic a bit so that checking None is not for python 2.7 only.
```suggestion
def _normalize_entity_path_to_full_path_if_needed(entity_path, fully_qualified_namespace):
    # type: (str, str) -> str
    if not entity_path:
        return entity_path
    parsed = urlparse.urlparse(entity_path)
    forward_to = "sb://" + fully_qualified_namespace + "/" + entity_pathif not parsed.netloc else entity_path
    return entity_path
```